src/adapter/src/coord/sequencer/inner/peek.rs

```diff
@@ -315,7 +324,17 @@ impl Coordinator {
                     optimizer_config,
                 ))
             }
-            Some(copy_to_ctx) => {
+            Some(mut copy_to_ctx) => {
+                // Getting the max worker count across replicas
+                // and using that value for the number of buckets to
+                // divide the copy to output into.
+                let max_worker_count = cluster
```

Comment:
cc @benesch Updated to use the max worker count across replicas as the number of buckets.